jenkins-pipelines/longevity-all-compressions-3days.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+longevityPipeline(
+    backend: 'aws',
+    aws_region: 'eu-west-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'longevity-all-compressions-3days.yaml',
```

Comment:
need to mention the full path
`test-cases/longevity/longevity-all-compressions-3days.yaml`